integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4331MissingJSONEmptyArrayNHTTPTransport.java

```diff
@@ -81,8 +80,7 @@ public void testJSONEmptyArrayMissingNHTTPTransport() throws Exception {
 
     @AfterClass(alwaysRun = true)
     public void destroy() throws Exception {
-        super.cleanup();
-        serverConfigurationManager.restoreToLastConfiguration();
+        serverConfigurationManager.restoreToLastMIConfiguration();
```

Comment:
Shall we group all these NHTP test together and re start the server only once.

Reply:
Done. Please review